src/reference/01-General-Info/90-Changes/45-sbt-0.13-syntax-migration-guide.md

```diff
@@ -0,0 +1,105 @@
+---
+out: sbt-0.13-Syntax-Migration-Guide.html
+---
+
+## sbt 0.13 Syntax Migration Guide
+
+### Introduction
+
+Before sbt 0.13 (sbt 0.9 to 0.12) it was very common to see in builds the usage of two aspects of sbt:
+
+* the "fishbone" operator: `<<=`, `<+=`, `<++=`
```

Comment:
I don't want to introduce "fishbone" term here ourselves especially now that I'm getting rid of it.
If we're going to name it, I would call them "key dependency operator".